test/config/FeatureSwitchesSpec.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import play.api.Configuration
+import support.UnitSpec
+
+class FeatureSwitchesSpec extends UnitSpec {
+
+  "a feature switch" should {
```

Comment:
Why not test for a general case (and make isEnabled public)? Otherwise we will have to test all these combinations for all switches (and in reality a switch is only represented by its name). See how BSAS does it [here](https://github.com/hmrc/self-assessment-bsas-api/blob/0cf7d51f345d01a6212027ac434ab29252d2d114/test/shared/config/FeatureSwitchesSpec.scala#L26-L51).